pantheon-bundle/frontend/src/app/moduleDisplay.tsx

```diff
@@ -237,6 +237,8 @@ class ModuleDisplay extends Component<any, any, any> {
     }
 
     private getVersionUUID = (path) => {
+        // remove /module from path
+        path = path.substring(7)
```

Comment:
same as above comment

Reply:
I updated the code to make it more readable.